### ISSUE TITLE
Window is not getting maximized on macOS

### ISSUE THREAD

**Issue by FeodorFitsner** [flet-dev/flet#218]
This is macOS-specific issue, it works as expected on Windows.

Repro code:

```python
import logging
from time import sleep

import flet
from flet import Page, Text

logging.basicConfig(level=logging.DEBUG)

def main(page: Page):
    page.add(
        Text("Hello, world!")
    )
    sleep(1)
    page.window_maximized = True
    page.window_always_on_top = True
    page.update()

flet.app(target=main, port=8550, view=flet.FLET_APP)
```

**Comment by FeodorFitsner** [comment]
It looks like there is an issue in window_manager library on macOS: https://github.com/leanflutter/window_manager/issues/208

**Comment by FeodorFitsner** [comment]
It's been fixed in Flet 0.1.53.
